vcpkg/maintainers/handling-usage-files.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Supplying usage for ports
+description: Guidance for handling usage files.
+author: JavierMatosD
+ms.author: javiermat
+ms.date: 06/30/2023
+---
+# Suppyling usage for ports
+
+## Overview
+
+Providing usage documentation for ports allows users to easily adopt them in their
+projects. We highly encourage providing a `usage` file within the ports directory (`ports/<port
+name>/usage`) that describes the minimal steps necessary to integrate with a buildsystem.
+
+### Supplying a usage file
+
+To provide usage documentation put a `usage` text file in the port's `share`
+installation directory. The recommended approach is to call the `configure_file()` function in
```

Comment:
approach to what? To adding the usage file to the installation directory? If so, it'd be good to call it out, e.g. "To add a `usage` file to the port's `share` installation directory, call the `configure_file()` function..."  Will they know how to do that?